sdk/core/azure-core/inc/azure/core/response.hpp

```diff
@@ -31,8 +31,7 @@ template <class T> class Response final {
   std::unique_ptr<Azure::Core::Http::RawResponse> RawResponse;
 
   /**
-   * @brief Initialize an #Azure::Response<T> with the value and raw response returned by the
-   * service.
+   * @brief Constructs a `%Response` with the value and raw response returned by the service.
```

Comment:
Does this show the templated type correctly in doxygen, like `Response<T>`?

Reply:
It would print the name in monospace font without the link, i.e without the self-reference.

Reply:
Please LMK if you think we should be monospacing it as `Response<T>` (then it should apply to `Nullable<T>` as well). I don't think we should.

Reply:
So far, not taking this to a follow up PR.

Reply:
> monospacing it as `Response<T>`

What do you mean by that?

Reply:
Nevermind, I get what you mean by `monospacing`, the "code" like font. I was more curious about the template `<T>`, maybe we should explicitly add that whenever referring to the type in the docs. So, not `Response` but rather `Response<T>`

Reply:
See how it renders in the published artifacts - https://dev.azure.com/azure-sdk/public/_build/results?buildId=922992&view=artifacts&pathAsName=false&type=publishedArtifacts

Reply:
I.e. 
![image](https://user-images.githubusercontent.com/41349689/120699454-40a18400-c465-11eb-8f7a-528880fe9b5e.png)

Reply:
> > I don't think we should.
> I am curious, how come?

Doxygen refers to it as simply `Response`, not `Response<>` or `Response<T>`. `T` is also a type parameter name, although a very common one. What if we rename `T` to `ResponseType`, should we start referring to it as `Response<ResponseType>`? I think of `<T>` as an implementation detail. It could've been used to distinguish between different `Response` types, should've had a templated and not templated versions, but this is not the case.

Reply:
And if we'd have to emphasize it is a template, I think more pedantic way to call it would be `Response<>`, not `Response<T>`.

Reply:
Makes sense, I think we can leave it as you have it for now.